charts/consul/values.yaml

```diff
@@ -590,28 +590,6 @@ global:
     # @type: boolean
     enableGatewayMetrics: true
 
-  # For connect-injected pods, the consul sidecar is responsible for metrics merging. For ingress/mesh/terminating
-  # gateways, it additionally ensures the Consul services are always registered with their local Consul client.
-  # @type: map
-  consulSidecarContainer:
```

Comment:
Could you add this as a breaking change to changelog?

Reply:
Done!